virttest/utils_test/libvirt.py

```diff
@@ -1150,6 +1150,54 @@ def thread_func_migration(self, vm, desturi, options=None,
                 self.RET_MIGRATION = False
                 self.RET_LOCK.release()
 
+    def do_migration_asynch(self, vm, srcuri, desturi, options=None,
+                            ignore_status=False):
+        """
+        Migrate a vm in a thread. This enable to execute more operations
+        during migration.
+
+        :param vm: The guest to be migrated
+        :param srcuri: The source connection uri
+        :param desturi: The target connection uri
+        :param options: The options for migration command.
+        :param ignore_status: True, means no CmdError will be caught
+                              for the failure.
+                              False, means an CmdError will be caught
+                              for the failure.
+
+        :return: The migration thread object and its starting time
+        """
+        migration_thread = threading.Thread(target=self.thread_func_migration,
+                                            args=(vm, desturi, options,
+                                                  ignore_status))
+        stime = int(time.time())
+        migration_thread.start()
+        return (migration_thread, stime)
+
+    def check_migration_result(self, migration_thread, start_time,
+                               thread_timeout=60):
+        """
```

Comment:
I don't think it is suitable for using this func name, the function is used for waiting migration finish, so what about migration_thread_wait?